python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -365,7 +369,7 @@ def kill(self):
         if not self.id:
             return None
         dirac_cmd = 'kill(%d)' % self.id
-        result = execute(dirac_cmd)
+        result = execute(dirac_cmd, return_raw_dict=True)
```

Comment:
In these cases where you do
```python
result = execute(dirac_cmd, return_raw_dict=True)
if not result_ok(result):
    raise BackendError('Dirac', 'Could not kill job: %s' % str(result))
return result['OK']
```
could it not be instead
```python
try:
    result = execute(dirac_cmd)
except GangaDiracException as err:
    raise BackendError('Dirac', 'Could not kill job: %s' % err)
return True
```
which in this case (and I'd expect most all others) could remove the need for the `return_raw_dict` parameter for `execute` entirely. This PR seems to have added the ability to do automatic error checking by default for all `execute` calls but has then turned it off for almost all cases.